lib/vty.c

```diff
@@ -390,6 +390,18 @@ int vty_json_no_pretty(struct vty *vty, struct json_object *json)
 	return vty_json_helper(vty, json, JSON_C_TO_STRING_NOSLASHESCAPE);
 }
 
+
+void vty_json_key(struct vty *vty, const char *key, bool *first_key)
+{
+	vty_out(vty, "%s\"%s\":", *first_key ? "{" : ",", key);
+	*first_key = false;
+}
+
+void vty_json_key_close(struct vty *vty)
```

Comment:
Not sure about the naming here. Isn't this more like `vty_json_close()`?

Reply:
done